### DISCUSSION TITLE
#64 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0x-stan** [tog-ETH-er/togETHer#49] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/exb-ruaw-sdx

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

3个PR #964 ～ #966

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会
 
- 《Arbitrum Sequencer inbox/feed 消息解析》by JasonWan
  - Youtube: https://youtu.be/KKm8UBy7we0
  - Bilibili: https://www.bilibili.com/video/BV1ns4y1q7sZ/

- 《f(x) 项目解读》by Tao
  - Youtube: https://youtu.be/CVvDw3mwzps
  - Bilibili: https://www.bilibili.com/video/BV1Y8411o7o5/

**Comment by Stamp9** [comment]
## 原语里弄

研究进行中。

**Comment by luckyyang** [comment]
Rebase Community

- web3 极客日报: 最近 14 期 http://daily.rebase.network/
- 招聘: 3 个团队发布招聘 https://github.com/rebase-network/who-is-hiring/issues
- web3极客公开课（web3geekforge ）6-10 期 https://space.bilibili.com/382886213/video
- 4 个活动 https://github.com/rebase-network/event-pool/issues
